client/pyroclient/client.py

```diff
@@ -29,6 +29,7 @@
     "detections-create": "/detections",
     "detections-label": "/detections/{det_id}/label",
     "detections-fetch": "/detections",
+    "detections-fetch-unck": "/detections/unacknowledged/from",
```

Comment:
From is a bit confusing in the route url: I first wasn't sure whether it was "from camera ID" or something else

Open question about the exact need: 
- if we need the last 24h, let's not leave the option
- otherwise a time range or pagination could be useful?

Reply:
@MateoLostanlen 

I can juste write fromDate ?
(we cannot put nothing because FastAPI think we try do call the detections/ endpoints with the Id=unacknowledged

Reply:
Oh good point, how about (not sure about PascalCasing in URL):
`/detections/unlabeled/fromdate`